public/index.php

```diff
@@ -6,9 +6,11 @@
 chdir(dirname(__DIR__));
 
 // Decline static file requests back to the PHP built-in webserver
-if (php_sapi_name() === 'cli-server' && is_file(__DIR__ . parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH))) {
+$path = __DIR__ . parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH);
+if (php_sapi_name() === 'cli-server' && __FILE__ !== $path && is_file($path)) {
```

Comment:
Can `$path` be different from `__FILE__` and still be equivalent (symlink)? `realpath()` it?